scipy/signal/tests/test_signaltools.py

```diff
@@ -184,15 +183,15 @@ def test_convolve_method(self, n=100):
                 types.remove(dtype)
 
         args = [(t1, t2, mode) for t1 in types for t2 in types
-                               for mode in ['valid', 'full', 'same']]
+                for mode in ['valid', 'full', 'same']]
 
         # These are random arrays, which means test is much stronger than
         # convolving testing by convolving two np.ones arrays
         np.random.seed(42)
         array_types = {'i': np.random.choice([0, 1], size=n),
                        'f': np.random.randn(n)}
         array_types['b'] = array_types['u'] = array_types['i']
-        array_types['c'] = array_types['f'] + 0.5j*array_types['f']
+        array_types['c'] = array_types['f'] + 0.5j * array_types['f']
```

Comment:
this change is incorrect, the operator with higher precedence can be written without spaces